terraform/modules/app-ecs-services/templates/alertmanager.tpl

```diff
@@ -101,4 +105,7 @@ receivers:
   pagerduty_configs:
     - service_key: "${verify_p1_pagerduty_key}"
   slack_configs: *verify-2ndline-slack-configs
-
+- name: "verify-p2"
+  pagerduty_configs:
+    - service_key: "${verify_p1_pagerduty_key}"
```

Comment:
I don't think this is the right key - it will need to be a different service for p2

Reply:
I'll look into how to make one and where to put it.

Reply:
it lives in re-secrets and we have terraform pass provider data sources to pull secrets out of there